corehq/blobs/__init__.py

```diff
@@ -19,6 +19,11 @@ def get_blob_db():
     return _db[-1]
 
 
+def get_blob_db_exporter(slug, domain):
+    from .migratingdb import MigratingBlobDB
+    return MigratingBlobDB(_get_zip_db(slug, domain), get_blob_db())
```

Comment:
I'm pretty sure this should not be using `MigratingBlobDB` since the only operation called on this db is `copy_blob`, which should only affect the zip db.